dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowSubClass.java

```diff
@@ -1,10 +1,35 @@
 package io.dockstore.webservice.resources;
 
+import io.dockstore.webservice.core.AppTool;
+import io.dockstore.webservice.core.BioWorkflow;
+import io.dockstore.webservice.core.Notebook;
+import io.dockstore.webservice.core.Service;
+import io.dockstore.webservice.core.Workflow;
 import io.swagger.v3.oas.annotations.media.Schema;
 
 @Schema(enumAsRef = true)
 public enum WorkflowSubClass {
     BIOWORKFLOW,
     SERVICE,
-    APPTOOL
+    APPTOOL,
```

Comment:
you can do better by storing the target class in the enum itself 
https://github.com/dockstore/dockstore/blob/develop/dockstore-common/src/main/java/io/dockstore/common/DescriptorLanguage.java#L37 for examples